specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
FWIW, CDOS folks seemed to say on the call today that the Dominion BallotType field corresponded one-to-one to the subsets (of the set of all contests) that actually appeared on ballots in the election. The example they gave was that if there was only one contest in a particular election, and if it was a county-wide contest, then there would be only one BallotType. If this is not true, a counter-example would be helpful.